filters/include/pcl/filters/impl/crop_hull.hpp

```diff
@@ -104,81 +84,94 @@ pcl::CropHull<PointT>::getHullCloudRange ()
     -std::numeric_limits<float>::max (),
     -std::numeric_limits<float>::max ()
   );
-  for (std::size_t index = 0; index < indices_->size (); index++)
+  for (pcl::Vertices const & poly : hull_polygons_)
```

Comment:
Pull out as a separate PR please (repeating since I resolved the conversation)

Reply:
created separate PR (#3976) for issue #3960